R/pkg/R/SQLContext.R

```diff
@@ -584,7 +584,7 @@ tableToDF <- function(tableName) {
 #'
 #' @param path The path of files to load
 #' @param source The name of external data source
-#' @param schema The data schema defined in structType
+#' @param schema The data schema defined in structType or a DDL-formatted string.
```

Comment:
Ah... this is, actually and up to my knowledge, loosely related with https://issues.apache.org/jira/browse/SPARK-14764 ... I will keep in mind and add a link if this one is resolved.